docs/kubernetes/how-to-deploy-kubeflow-on-linode-kubernetes-engine/index.md

```diff
@@ -0,0 +1,252 @@
+---
+author:
+  name: Daniele Polencic
+  email: daniele@learnk8s.io
+description: 'How to deploy Kubeflow and train a model using Kubeflow pipelines on Linode Kubernetes Engine.'
+og_description: 'Learn how to deploy Kubeflow and train a model using Kubeflow pipelines on Linode Kubernetes Engine.'
+keywords: ['kubernetes','kubeflow','machine learning','containers','tensorflow']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2020-07-08
+modified_by:
+  name: Linode
+title: "How to deploy Kubeflow on Linode Kubernetes Engine"
+h1_title: "How to deploy Kubeflow on Linode Kubernetes Engine"
+contributor:
+  name: Daniele Polencic
+  link: https://github.com/danielepolencic
+external_resources:
+- '[Multi-user, auth-enabled Kubeflow installation](https://www.kubeflow.org/docs/started/k8s/kfctl-istio-dex/)'
+- '[Kubeflow Jupyter notebooks](https://www.kubeflow.org/docs/notebooks/)'
+- '[Kubeflow pipelines](https://www.kubeflow.org/docs/pipelines/)'
+---
+
+## What is Kubeflow?
+
+[Kubeflow](https://www.kubeflow.org/) is an excellent alternative for training and evaluating machine learning models in public and private clouds.
+
+Kubeflow is designed to make your machine learning experiments portable and scalable.
+
+You start by creating [Jupyter notebooks](https://jupyter.org/) in the cloud.
+
+When you sense that you have hit a breakthrough, you can scale your model to run on thousands of machines.
+
+Kubeflow optimises your model and breaks it down into smaller tasks that can be processed in parallel.
+
+Then, it distributes the tasks to several computers and waits until the results are ready.
+
+_Are you ready to train your model at scale?_
+
+## Before You Begin
+
+1.  Please notice that you can run this tutorial locally using [minikf](https://www.kubeflow.org/docs/started/workstation/getting-started-minikf/). However, you should have at least 12GB of RAM, 2CPUs and 50GB of disk space available.
+
+2.  The `kfctl` CLI used to deploy Kubeflow currently only works on Linux and Mac. If you're on Windows, you can use WSL2 or a Docker container to work around this limitation.
+
+3.  Most Kubeflow pipelines require [Kubernetes Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes) that can be attached to several nodes at once (ReadWriteMany). Currently, the only mode supported by the Linode Block Storage CSI driver is ReadWriteOnce, meaning that it can only be connected to one Kubernetes node at a time.
+
```

Comment:
Caution

    This guide’s example instructions create several billable resources on your Linode account. If you do not want to keep using the example cluster that you create, be sure to delete it when you have finished the guide.

    If you remove the resources afterward, you will only be billed for the hour(s) that the resources were present on your account. Consult the Billing and Payments guide for detailed information about how hourly billing works and for a table of plan pricing.

Reply:
Amended